package.json

```diff
@@ -6,7 +6,7 @@
   "main": "./dist/main/mocha.js",
   "module": "./dist/main/mocha.js",
   "bin": {
-    "mocha-parallel-tests": "dist/bin/cli.js"
```

Comment:
https://stackoverflow.com/questions/50616253/how-to-resolve-chicken-egg-situation-with-tsc-and-npm-install